.github/workflows/build.yml

```diff
@@ -320,6 +333,17 @@ jobs:
         run:
           cp LICENSE ./control-plane
       - uses: hashicorp/actions-docker-build@v1
```

Comment:
Is this step empty now?

Reply:
I guess, weird, this wasn't a change I made

Reply:
oh maybe I did... one sec

Reply:
But a new step with that action was added after the fact. Probably, there was a conflict here.

Reply:
yeah, I didn't get a conflict, I guess this is how it merged the cherry pick

Reply:
Ok, I think I just eliminate this uses, and the docker build gets done down lower

Reply:
that was a weird diff 😅

Reply:
@DanStough should be fixed